randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -672,15 +675,22 @@ Extra - asset_id: 92
   * Layers: default
   * Normal Door to Big Pink/Door to Green Hill Zone
   > Door to Morph Ball Room
-      Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+      Any of the following:
+          Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+          # IBJ
+          Vertical Infinite Bomb Jump (Intermediate) and Can Lay Bombs
   > Door to Noob Bridge
       Trivial
   > Pickup (Missile Expansion)
       All of the following:
```

Comment:
Can you comment the ANY blocks as something like "from pipe to pickup/reach pipe" or something?